src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -58,6 +59,9 @@ public class DefaultConfiguration : IConfiguration
         private readonly RunTimeConfiguration _runTimeConfiguration = new RunTimeConfiguration();
         private readonly SecurityPoliciesConfiguration _securityPoliciesConfiguration = new SecurityPoliciesConfiguration();
         private Dictionary<string, string> _newRelicAppSettings { get; }
+        private bool _hasObscuringKey;
+        private bool _hasEncryptedPassword;
```

Comment:
See below, I don't think we need these.  We only use them in a getter only one time.